File: AnalisisNumerico-master/Problemas Python/Raiz.py
import math

def raiz(n, E, x):
    y = float(0)
    contador = 0
    while(math.fabs(x-y) > E):
        y = n/x + x
        y = y/2
        x = y
        contador += 1
    return (y, contador)


(result, it) = raiz(7.0, 0.01, 2)
print("Se obtiene el resultado de ", result, ". Con ", it, " iteracciones")

File: Problemas R Taller1/Raiz.R
rm(list=ls())

Raiz = function(n, E, x, y)
{
  contador = 0
  while (abs(x -y) > E) {
    y = (x + n/x)
    y = y / 2
    x = y
    contador = contador + 1
  }
  return (c(y, contador))
}
resultado= Raiz(7, 0.1, 3, 0)
cat("El resultado de la raiz es: ", resultado[1], "Con: ", resultado[2], "iteraccion \n")